auto3dseg/tests/test_algo_templates.py

```diff
@@ -48,22 +48,19 @@
 
 sys.path.insert(0, algo_templates)
 
-num_epochs = 1
 n_data = 8
```

Comment:
`n_data` is no longer needed as we don't need to calculate the iterations

Reply:
just removed it.